<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
        <meta name="description" content="" />
        <meta name="author" content="" />
        <title>Resume - Manoj Kumar</title>
        <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />
        <!-- Font Awesome icons (free version)-->
        <script src="https://use.fontawesome.com/releases/v6.1.0/js/all.js" crossorigin="anonymous"></script>
        <!-- Google fonts-->
        <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet" type="text/css" />
        <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
        <!-- Core theme CSS (includes Bootstrap)-->
        <link href="css/styles.css" rel="stylesheet" />
    </head>
    <body id="page-top">
        <!-- Navigation-->
        <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
            <a class="navbar-brand js-scroll-trigger" href="#page-top">
                <span class="d-block d-lg-none">Manoj Kumar</span>
                <span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="assets/img/manoj_pic" alt="..." /></span>
            </a>
            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
            <div class="collapse navbar-collapse" id="navbarResponsive">
                <ul class="navbar-nav">
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#about">About</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#experience">Professional-Preparation</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#education">Education</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#publication">Research</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="projects.html">Projects</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#skills">Computing Skills</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#wet-lab">WET-LAB Skills</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#grant">GRANTS AND FUNDING</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#teaching">Teaching</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#awards">Blogs</a></li>
                </ul>
            </div>
        </nav>
        <!-- Page Content-->
        <div class="container-fluid p-0">
            <!-- About-->
            <section class="resume-section" id="about">
                <div class="resume-section-content">
                    <h1 class="mb-0">
                        Manoj
                        <span class="text-primary">Kumar</span>
                    </h1>
                    <div class="subheading mb-5">
                        ADL · PSU, PA 16802 ·
                        <a href="mailto:sekhwal@gmail.com">sekhwal@gmail.com</a>
                    </div>
                    <p class="lead mb-5">
                    	• A detail-oriented and analytical professional with a Ph.D.
			• 5+ years total postdoctoral experience working in bioinformatics, genomics, GWAS, RNASeq analysis and metagenomics.
			• Extensive experience in analyzing data, developing research proposals, writing research articles, and preparing reports 					in detailed and scientific manner.
			• Excellent in designing experiments, models and analytical skills. 
			• Experience in managing and developing bioinformatics lab and supervising graduates and post-graduates.</p>
                    <div class="social-icons">
                        <a class="social-icon" href="https://www.linkedin.com/in/manoj-kumar-ph-d-606725146/"><i class="fab fa-linkedin-in"></i></a>
                        <a class="social-icon" href="https://github.com/sekhwal?tab=repositories"><i class="fab fa-github"></i></a>
                        <a class="social-icon" href="https://www.researchgate.net/profile/Manoj-Sekhwal"><i class="fab fa-researchgate"></i></a>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Experience-->
            <section class="resume-section" id="experience">
                <div class="resume-section-content">
                    <h2 class="mb-5">Professional-Preparation</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Postdoctoral Researcher</h3>
                            <div class="subheading mb-3">Department of Veterinary and Biomedical Sciences, Pennsylvania State
				University, USA</div>
                            <p>-Characterization microbial genomes and genome assembly, microbial GWAS, population genomics, next-generation 					sequencing, bioinformatics approaches for developing and validating innovative diagnostic tests and
				surveillance platforms for bacterial and viral pathogens.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">September 2021 - Present</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Postdoctoral Researcher </h3>
                            <div class="subheading mb-3">Northern Arizona University, USA</div>
                            <h6><i>Major tasks accomplished:</i></h6>
                            <p>-GWAS, population genetics, environmental association study of Giant Sequoia and Costal
			     Redwood.</p>
			     <h6><i>Major pipelines/methods used:</i></h6>
			     <p>-GWAS/population genomics: GATK, BEDtools, BCFtools, VCFtools, TASSEL, GEMMA, PLINK,
				fastStructure, GAPIT, CLUMPAK <br>
				-RNA Seq analysis: FastQC, MultiQC, TrimGalore, Cutadaptor, Bowtie2, HISAT2, STAR, BLASTP,
				DESeq2, IGV  <br>
				-Functional Analysis: pfam, BlastKOALA, HMMER, blastp, KEGG, Blast2GO, BINGO-Cytoscape,
				NCBI, GO enrichment  <br>
				-R Packages/Analysis: pophelper, adegenet, popGenome, hierfstat, vegan, tidyverse, qqman,
				ggplot2, RDA, LD analysis, LFMM  <br>
				-Linux/HPC: NAU Monsoon (high-performance computer cluster), Ubuntu  <br>
				</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">February 2021 - August 2021</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Postdoctoral Research Fellow</h3>
                            <div class="subheading mb-3">North Dakota State University, USA</div>
                            <h6><i>Major tasks accomplished:</i></h6>
                            <p>-Genetic variants identification of specific phenotypes using GWAS to understand the
				evolutionary adaption and ecological divergence in 277 E. coli genomes. <br>
				-Illumina HiSeq 4000 sequencing data assembly, annotation and pangenome analysis <br>
				-16S, ITS microbiome analysis, minION data <br>
				-RNASeq data analysis in order to identify differential gene expression and pathways analysis. <br>
				</p>
			    <h6><i>Major pipelines/methods used:</i></h6>
                            <p>-GWAS Study: SPAdes, Prokka, ROARY, RAxML, DBGWAS, BEDtools, blastn, Cytoscape, Geneious <br>
				-RNA Seq analysis: sratoolkit, FastQC, MultiQC, TrimGalore, Cutadaptor, SOAPnuke, HISAT2,
				Samtools, DESeq2, IGV, minimap2, longshot <br>
				-Metagenomics: moving-pictures-tutorial at qiime2, SILVA, MetONTIIME, 16S rRNA, ITS. <br>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">(April 2019 - June 2020</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Visiting Fellow, NSERC</h3>
                            <div class="subheading mb-3">Agriculture and Agri-Food Canada</div>
                             <h6><i>Major tasks accomplished:</i></h6>
                            <p>-Composed pipelines for identifying RGAs, pseudogenes and marker genes from the whole
				genome of wheat, flax and Arabidopsis.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary"> January 2015 - 2016</span></div>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Education-->
            <section class="resume-section" id="education">
                <div class="resume-section-content">
                    <h2 class="mb-5">Education</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Banasthali University, India</h3>
                            <div class="subheading mb-3">-Ph.D.</div>
                            <div>Major : Bioinformatics</div>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">Sept. 2009 - Feb. 2015</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Sikkim Manipal University, India</h3>
                            <div class="subheading mb-3">M.Sc</div>
                            <div>Major : Bioinformatics</div>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">2003 - 2006</span></div>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Publication-->
            <section class="resume-section" id="publication">
                <div class="resume-section-content">
                  <h2 class="mb-5">Publications</h2>
                    <p>
                    
                    [1] Neale, D.B., Zimin, A.V., Zaman, S., Scott, A.D., Shrestha, B., Workman, R.E., Puiu, D.,
			Allen, B.J., Sekhwal, M.K., et al. 2022. Assembled and annotated 26.5 Gbp coast redwood
			genome: a resource for estimating evolutionary adaptive potential and investigating
			hexaploidy origin. G3: Genes Genomes Genetics, 12: 1-13.<br>
			[2] De La Torre, A.R., Sekhwal, M.K., et al. 2022. Genome-wide association identifies
			candidate genes for drought tolerance in coast redwood and giant sequoia. The Plant
			Journal. 109: 7-22.<br>
			[3] De La Torre, A.R., Sekhwal, M.K., Neale, D.B. 2021. Selective sweeps and polygenic
			adaptation drive local adaptation along moisture and temperature gradients in natural
			populations of coast redwood and giant sequoia. Genes, 12(11):1826.<br>
			[4] Xiao, J., Sekhwal, M.K., et al., 2016. Pseudogenes and their genome-wide prediction in
			plants. Intl. J. Mol. Sci., Int. J. Mol. Sci. 17: 1991-2006.<br>
			[5] Sekhwal, M.K., et al., 2015. Disease resistance gene analogs (RGAs) in plants. Intl. J. Mol.
			Sci., 16: 19248-19290.<br>
			[6] Sekhwal, M.K., et al., 2015. Identification of drought induced transcription factors in
			Sorghum bicolor using go-terms semantic similarity. Cell Mol Biol Lett. 20: 1-23.<br>
			[7] Sekhwal, M.K., et al., 2013. Annotation of glycoside hydrolases in Sorghum bicolor using
			proteins interaction approach. J Proteome Sci Comput Biol. 2: 2.<br>
			[8] Sekhwal, M.K., et al., 2013. Identification of MFS proteins in sorghum using semantic
			similarity. Theory Biosci 132: 105-113.<br>
			[9] Sharma, V., Sekhwal, M.K., 2012. Identification of drought responsive proteins using
			gene ontology hierarchy. Bioinformation 8: 595–599.<br>
			[10] Sekhwal, M.K., et al., 2012. Identification of salt treated proteins in sorghum using gene
			ontology linkage. Physiol Mol Biol Plants 18: 209-216.<br>
			[11]. Sekhwal, M.K., Petersen, M.L., Hewitt, A.J., Schmidt, K.N., Bergholz, P.W., 2020. Comparison
			of genome-wide associations with fourteen phenotypes reveals adaptive variants that have
			generalized value for growth improvement (Draft version).
		   <div class="subheading mb-3">BOOK CHAPTER:</div>
                    <p>
                    [1] Weiss, M., Sekhwal, M.K., Neale, D.B., De La Torre, A.R. 2022. Genomics of climate
			adaptation in Pinus Lambertiana. The Pine Genomes. 51-65. 
		    </p>
		    <div class="subheading mb-3">POSTERS AND ABSTRACTS:</div>
                    <p>
                    [1] Sekhwal, M.K., Petersen, M.L., Hewitt, A.J., Schmidt, K.N., Bergholz, P.W. Comparison of
			genome-wide associations with fourteen phenotypes reveals adaptive variants that have
			generalized value for growth improvement. ASM 79th annual meeting, October 25-26, 2019.
			Duluth, MN, USA. <br>
		     [2] Sekhwal, M.K., Knight, D., Adl, S. Free living N2 fixation in Saskatchewan soils. 2nd
			International Symposium on Innovations in Plant and Food Sciences, September 8-10, 2017,
			UofS, SK, Canada. <br>
		    </p>
		    <div class="subheading mb-3">GENBANK SUBMISSION:</div>
                    <p>
                    [1] Sekhwal,M.K., Byukusenge,M., Jayarao,B.M. and Li,L. Rhodococcus equi strain P2120831
			chromosome, complete genome. 2022. Animal Diagnostic Lab, Penn State University.
			https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP093554.1.<br>
		     [2] Sekhwal,M.K., Byukusenge,M., Jayarao,B.M. and Li,L. Rhodococcus equi strain P2117036
			chromosome, complete genome. 2022. Animal Diagnostic Lab, Penn State University.
			https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP093553.1<br>
		    </p>
		    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Skills-->
            <section class="resume-section" id="skills">
                <div class="resume-section-content">
                    <h2 class="mb-5">Computing Skills</h2>
                    <p>
                    <div class="subheading mb-3">Programming Languages & Tools</div>
                    
                    • All my R coding exercises are available at <a href="#">Rpubs </a> <br> <br>
                    • All my python coding exercises are available at <a href="https://www.kaggle.com/kumarsekhwal/code"> Kaggle </a> <br> <br>
                    </p>
                    
                    <div class="subheading mb-3">Database developed</div>
                    
                   	• GWAS|k-mers (A database for CRISPR editing) using MySql, Apache HTTP Server, PHP, HTML-CSS at
			 Department of Microbiological Sciences, North Dakota State University, USA (2019 - 2020). <br> <br>
			 
			• T2D|Dbase and OncoGenDB databases developed using using MySql, Apache Server, PHP, HTML-CSS. (year 2010).
			 
                </div>
            </section>
            <hr class="m-0" />
            <!-- Wet-lab-->
            <section class="resume-section" id="wet-lab">
                <div class="resume-section-content">
                    <h2 class="mb-5">Wet-lab Skills</h2>
                   <p>
                       • DNA, RNA extraction <br>
			• DNA quantification using nanodrop <br>
			• Protein extraction <br>
			• Illumina MiSeq, 2D gel-electrophoresis <br>
                    
                    </p>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Grant-->
            <section class="resume-section" id="grant">
                <div class="resume-section-content">
                    <h2 class="mb-5">GRANTS AND FUNDING</h2>
                   <div class="subheading mb-3">DST Government of Rajasthan, India</div>
                   <p>
                   	<h6>Funded projects title:</h6>
                       • Phylogenetic profile of acetyl-co-enzyme a carboxylase for biodiesel production. year- 2010.<br>
			• Molecular docking and virtual screening for glycogen phosphorylase in diabetes. year - 2010.<br>
			• Tool developed for finding pattern repeats of the sequences. year-  2010.<br>
			• Software development for similar or non similar amino acid pattern repeats. year - 2009 <br>
                    
                    </p>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Teaching-->
            <section class="resume-section" id="teaching">
                <div class="resume-section-content">
                    <h2 class="mb-5">TEACHING</h2>
                    <h6>Delivered lectures as a guest speaker and conducted training programs on various topics of bioinformatics</h6>
			• Maharishi Dayanand (PG) College, Ganganagar, Rajasthan, India. year 2007.<br>
			• G.V.M Girls College, Sonipat, Haryana, India.  year 2008-2009.<br>
			• Biyani Girls College, Jaipur, India. year 2008-2009. <br>   <br>                
                </div>
            </section>
        </div>
        <!-- Bootstrap core JS-->
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
        <!-- Core theme JS-->
        <script src="js/scripts.js"></script>
    </body>
</html>
